### DISCUSSION TITLE
Is there a way to enable `nvim-cmp` or `blink` based on specified condition?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#5187] [Q&A]
Because I found out that blink doesn't work in Windows10, but it does in Windows11.

So I want to add a configuration to enable a different configuration on a disconnected system.

**Comment by dpetka2001** [comment]
You could maybe use `.lazy.lua` for that purpose?? Not really sure if it fits your needs but you could give it a try.
